### PR TITLE
schedule breadcrumbing to filter venue specific events

### PR DESCRIPTION
**Pull request by malaikaaiyar**
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/1002

### CODE REVIEW

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 1]
```diff
@@ -243,13 +258,19 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       scheduleDate: day,
       daysEvents: showPersonalisedSchedule
         ? eventsFilledWithPriority.filter((event) => event.isSaved)
+        : filterRelatedEvents
+        ? eventsFilledWithPriority.filter(
+            (event) => event.room === venueRoomTitle
+          )
```
Comment (yurii-lubynets) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:264]: let's avoid nested ternar operators. It's really hard to read

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 2]
```diff
@@ -282,6 +303,30 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   return (
     <div className="NavBarWrapper">
       <div className={containerClasses}>
+        <div className="NavBarSchedule__breadcrumb">
+          <button
+            onClick={() => setFilterRelatedEvents(false)}
+            className={filterRelatedEvents ? "button--a disabled" : "button--a"}
```
Comment (yurii-lubynets) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:309]: you could use `classnames` package to manage styles

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 3]
```diff
@@ -282,6 +303,30 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
+          </button>
+
+          {venueId !== sovereignVenue?.id && (
+            <>
+              {" "}
+              /{" "}
+              <button
+                onClick={() => setFilterRelatedEvents(true)}
+                className={
+                  !filterRelatedEvents ? "button--a disabled" : "button--a"
```
Comment (yurii-lubynets) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:321]: Is it actually right to add "button" classname to `<button/>` tag. 
It's worth keeping more sense here

**Review by yurii-lubynets — COMMENTED** [review]
Looks nice for me, thank you for your work. Some comments left

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 3 above)
Comment (malaikaaiyar) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:321]: if i add the "button" classname it styles it like a button, whereas "button--a" styles it as a link

**Pull request closed without merging**